### ISSUE TITLE
a11y: insufficient color contrast on h2 "Gift Styles" - profile page

### ISSUE THREAD

**Issue by Danielle254** [LetsGetTechnical/elecretanta#521]
'/profile' page
h2 "Gift Styles" text has insufficient color contrast to meet WCAG 2.1 AA standards

![Image](https://github.com/user-attachments/assets/af92624d-9bab-4b42-919e-c4f60bb95bd9)
